src/plugins/circular_view/src/CircularView.h

```diff
@@ -45,30 +45,31 @@
 
 namespace U2 {
 
-class CircularViewRenderArea;
 class CircularAnnotationItem;
 class CircularAnnotationLabel;
 class CircularAnnotationRegionItem;
+class CircularViewRenderArea;
 struct CircularViewSettings;
 class TextItem;
 
 class CircularView : public GSequenceLineViewAnnotated {
     Q_OBJECT
 public:
     CircularView(QWidget *p, ADVSequenceObjectContext *ctx, CircularViewSettings *settings);
-    void pack();
-    void mousePressEvent(QMouseEvent *e);
-    void mouseMoveEvent(QMouseEvent *e);
-    void mouseReleaseEvent(QMouseEvent *e);
-    void resizeEvent(QResizeEvent *e);
 
-    void keyPressEvent(QKeyEvent *e);
-    void keyReleaseEvent(QKeyEvent *e);
+    void pack() override;
+    void mousePressEvent(QMouseEvent *e) override;
+    void mouseMoveEvent(QMouseEvent *e) override;
+    void mouseReleaseEvent(QMouseEvent *e) override;
+    void resizeEvent(QResizeEvent *e) override;
+
+    void keyPressEvent(QKeyEvent *e) override;
+    void keyReleaseEvent(QKeyEvent *e) override;
 
-    void wheelEvent(QWheelEvent *we);
-    virtual QSize sizeHint() const;
+    void wheelEvent(QWheelEvent *we) override;
+    virtual QSize sizeHint() const override;
```

Comment:
Usually you remove "virtual" word in that cases.

Reply:
Done